.vitepress/theme/components/ShareFeedback.vue

```diff
@@ -0,0 +1,252 @@
+    kind: feedbackKind.value,
+    feedback: feedbackText.value,
+    contact: contact.value,
+    location: window.location,
+  }
+  const api = mande(props.feedbackUrl)
+  action.set(api.post(data))
+}
+</script>
+
+<template>
+  <SButton
+    type="primary"
+    size="sm"
+    @click="openModal = true"
+  >
+    <template #icon>
+      <IconFeedback class="-mb-2px" />
+    </template>
+    Share feedback
+  </SButton>
+
+  <SModal
+    v-slot="api"
+    v-model:show="openModal"
+    described-by="share-feedback-description"
+    @after-close="onAfterClose"
+  >
+    <div class="feedback-card shadow-lg flex flex-col">
+      <div class="feedback-card_header flex items-center">
+        <div
+          :id="api.labelledBy"
+          class="feedback-card_title flex-1"
+        >
+          Share feedback
+        </div>
+
+        <SButton
+          size="sm"
+          type="action"
+          @click="api.close()"
+        >
+          <template #icon>
+            <IconClose />
+          </template>
+        </SButton>
+      </div>
+
+      <template v-if="success">
+        <div class="p-4 flex items-center space-x-4">
+          <IconCheck class="text-3xl feedback-card_check" />
+          <div>Thank you for sharing your feedback!</div>
+        </div>
+
+        <div class="flex flex-row-reverse p-4">
+          <SButton @click="api.close()">
+            Close
+          </SButton>
+        </div>
+      </template>
+
+      <div
+        v-else
+        class="flex-1 overflow-y-scroll"
+      >
+        <div class="p-4 space-y-4">
+          <p
+            id="share-feedback-description"
+            class="text-sm"
+          >
+            Please take a moment to help us improve the Iroha 2 Documentation. We take your input very seriously.
+          </p>
+
+          <div>
+            <fieldset class="space-y-1">
+              <legend class="field-label">
+                What kind of feedback?*
+              </legend>
+
+              <div
+                v-for="value of KINDS"
+                :key="value"
+                class="flex space-x-2 items-center"
+              >
+                <input
+                  :id="`feedback-kind-${value}`"
+                  v-model="feedbackKind"
+                  class="max-w-min"
+                  :value="value"
+                  type="radio"
+                  name="feedback-kind"
+                >
+                <label
+                  :for="`feedback-kind-${value}`"
+                  class="flex-1 text-sm"
+                >{{ KINDS_LABELS[value] }}</label>
+              </div>
+            </fieldset>
+          </div>
+
+          <div>
+            <label
+              for="feedback-input-text"
+              class="field-label"
+            >Feedback*</label>
+
+            <textarea
+              id="feedback-input-text"
+              v-model="feedbackText"
+              placeholder="What can we do to improve the overall documentation browsing experience?"
```

Comment:
Do we need this placeholder? It doesn't really work with the bug type

Reply:
We don't really need it, but some placeholder is a good practice. I can make it dynamic depending on what option is selected, btw.

Reply:
Dynamic is a good idea.

Reply:
Okay. What text should I use?

Reply:
for bugs:
> Report any bugs or issues you found in Iroha 2 documentation.

Reply:
Done.